roles/vdm/tasks/postgres/postgres-instance.yaml

```diff
@@ -122,6 +122,45 @@
 
 - name: postgres instance - internal post 2022.10
   block:
+  - name: postgres instance - crunchy v4 to crunchy v5 update is required
+    fail:
+      msg: "A manual update from crunchy v4 to v5 is required, follow these steps https://go.documentation.sas.com/doc/en/itopscdc/default/dplynotes/p1bpcvd3sr8au8n1w9ypcvu31taj.htm"
+    when:
+      - v4_crunchy_present
```

Comment:
This seems like the more logical place for the `v4_crunchy_present` flag to be defined though the new name would be `v4_crunchy_found`

Reply:
adding is defined test prior to truthiness test in the when clause, will no longer need the default value set in line 6 of pg_config.yaml above, use v4_crunchy_found for flag name